Eventing docs updates for 6.5 (#1477)
couchbase/docs-server: modules/eventing/pages/eventing-faq.adoc

* Can the logic in a Function in production be altered while it is running?
+
Yes.
Functions can be "paused, edited and resumed" without losing a single mutation; as such continuity is maintained. Note this sequence is similar to the operations of "undeploy, edited, and deployed with a feed boundary ‘From now’", however in this later case you can lose mutations.

> * How to perform Functions lifecycle operations from CI/CD?

Reviewer: ```suggestion
* How do I perform a Function's lifecycle operations from CI/CD?
```

Grammar check

Yes.
For example, you can your enrich or update a document with data from another document (using a document id) from any other bucket that is exposed to the Function via a binding with access level of "Read Write" inclusive of the source bucket.

== Timer Behavior

> * Timer Delays: When I schedule a timer to fire at an exact time, I see some delay why?

Reviewer: ```suggestion
* Timer Delays: When I schedule a timer to fire at an exact time, I see some delay. Why?
```

+
The timer implementation is designed to handle large numbers of distributed timers (i.e., millions of timers) and the only promise is to run timers as soon as possible, e.g. no timers lost.
+
In a steady state you may see a 3-4 second delay from the scheduled time, however if scheduling timers close to the system wall-clock this delay may increase to about 14 seconds.  
For more details on Timer scheduling refer to xref:eventing-timers.adoc#wall-clock-accuracy[Timers: Wall-clock Accuracy] section.

> * Can I Cancel a Timer 

Reviewer: ```suggestion
* Can I cancel a timer? 
```

Question and title case for consistency

+
No this is not supported in 6.5, however this will be supported in 6.6.0 - refer to xref:eventing-timers.adoc#limitations[Timers: Limitations].

> * Can I create a Recurring timer 

Reviewer: ```suggestion
* Can I create a Recurring timer?
```

Question for consistency

+
No this is not supported in 6.5, however this will be supported in 6.6.0 -  xref:eventing-timers.adoc#limitations[Timers: Limitations].

> * Can I schedule Timer far into the future

Reviewer: ```suggestion
* Can I schedule a Timer far into the future?
```

Question for consistency

when the Eventing rebalance operation is in progress, you cannot perform a Function 
lifecycle operation.

* How do I increase performance of an Eventing Function?
+
You can scale up vertically by adding additional workers (in the handler's settings) to increase performance  
for a specific Function. 
You can also scale out horizontally via Couchbase’s elastic scaling option by adding another node and rebalancing.  In this case each eventing node is assigned a subset of vBuckets note this sharding increases overall performance for all Functions.
+

> However keep in mind that sometimes the Function is limited by the overall performance of the Data Service in this case it is appropriate to scale the Data service.

Reviewer: ```suggestion
However keep in mind that sometimes the Function is limited by the overall performance of the Data Service. In this case it is appropriate to scale the Data service.
```
